github_pr.py

```diff
@@ -441,9 +441,18 @@ def main():
     parser.add_argument('--condition-approved-mergers', default=None, nargs='+', help='list of usernames of approved mergers')
     parser.add_argument('--condition-approved-mergers-file', action='store_true', help='check a file for list of usernames of approved mergers')
     parser.add_argument('--approved-mergers-file-path', default='./MAINTAINERS.txt', help='location of file of usernames of approved mergers')
-
+    parser.add_argument("-v", "--verbose", const=1, default=0, type=int, nargs="?",
+                    help="increase verbosity: 0 = only warnings, 1 = info, 2 = debug. No number means info. Default is no verbosity.")
     args = vars(parser.parse_args())
 
+    logger = logging.getLogger()
```

Comment:
the way i saw it online was like this, but im not sure im doing it 100% correct. I thought i needed to instantiate a logging object (logger) to mod the levels for the numbers. Then i just use the logging the same as i would if it was just the logging lib. any input is appreciated. I did remove it and the code broke :)